unit_tests/testBinaryTrees/testBinaryTrees.f90

```diff
@@ -83,6 +83,8 @@ PROGRAM testBinaryTrees
   WRITE(*,*) '==================================================='
   WRITE(*,*) 'TESTING BINARY TREES PASSED!'
   WRITE(*,*) '==================================================='
```

Comment:
Get rid of this write statement while you're at it?

Reply:
yep